Source/EasyNetQ.Serialization.Tests/EasyNetQ.Serialization.Tests.csproj

```diff
@@ -1,26 +1,17 @@
 <Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="../Tests.props" />
 
-    <PropertyGroup>
-        <TargetFramework>net5.0</TargetFramework>
+  <PropertyGroup>
+    <TargetFramework>net5</TargetFramework>
```

Comment:
Net5 is just a shorter synonym.

Reply:
Yes I knew but 😀

Reply:
All other projects use `net5` now so it's OK.

Reply:
> with all the abstraction extrapolated, so **every** interfaces, like IMessage, IBus, etc...

I did not have such a goal at least now. Keep in mind that now it will most likely fail due to the fact that there is no clear separation of the interface part of the implementation. Interfaces often use quite specific classes with their dependencies, for example, `MessageProperties` which references underlying RabbitMQ.Client dll. Refactoring code is required. This is quite fulfilling the task, but it is not in my plans. My goal was to prepare the basis for this on the example of serializers. I will make a separate PR for small edits from review.

Reply:
See #1276

Reply:
I will rename this PR after merging #1276.

Reply:
Done.